### PR TITLE
feat: pulling info from db for settings page volunteers

### PR DESCRIPTION
**Pull request by jxmoose**
[//]: # "Feel free to customize this template to your project's vibes"

### :croissant: Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

### :baguette_bread: Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

[image placeholder]: 

## :sandwich: How to review
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'

## :pie: Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

### :pretzel: Online sources
[//]: # 'Copy links to any tutorials or documentation that was useful to you when working on this PR'

### :bagel: Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @jxmoose

### CODE REVIEW

File: app/settings/styles.ts [hunk 1]
```diff
@@ -25,3 +28,37 @@ export const SettingDiv = styled.div`
...
+export const Email = styled(P)`
+  color: ${COLORS.gray10};
+  font-style: normal;
+  font-weight: 400;
+  margin-top: 0.25rem;
+`;
+
+export const SignOutButton = styled.button`
+  border: 1px solid ${COLORS.pomegranate12};
+  padding: 0.5rem 1rem;
```
Comment (jxmoose) [inline app/settings/styles.ts]: this second padding value should be .75 rem

File: components/SettingsCard/SettingsCardPerformanceInterest.tsx [hunk 2]
```diff
@@ -22,54 +22,40 @@ export default function SettingCardPerformanceInterest() {
...
-                    <styles.TruncatedText
-                      $fontWeight="400"
-                      $color={COLORS.gray11}
-                      $align="left"
-                    >
-                      Music
-                    </styles.TruncatedText>
-                  </li>
-                </styles.SettingListedItems>
+              <ul style={{ paddingLeft: '1.25rem', listStyleType: 'disc' }}>
```
Comment (jxmoose) [inline components/SettingsCard/SettingsCardPerformanceInterest.tsx]: use the SettingListedItems styled component

File: components/SettingsCard/SettingsCardPerformanceInterest.tsx [hunk 3]
```diff
@@ -22,54 +22,40 @@ export default function SettingCardPerformanceInterest() {
...
-                    <styles.TruncatedText
-                      $fontWeight="400"
-                      $color={COLORS.gray11}
-                      $align="left"
-                    >
-                      Country
-                    </styles.TruncatedText>
-                  </li>
-                </styles.SettingListedItems>
+              <ul style={{ paddingLeft: '1.25rem', listStyleType: 'disc' }}>
```
Comment (jxmoose) [inline components/SettingsCard/SettingsCardPerformanceInterest.tsx]: use the SettingListedItems styled component

File: components/SettingsCard/SettingsCardShowPreferences.tsx [hunk 4]
```diff
@@ -22,63 +22,60 @@ export default function SettingCardShowPreferences() {
...
-                    <styles.TruncatedText
-                      $fontWeight="400"
-                      $color={COLORS.gray11}
-                      $align="left"
-                    >
-                      Senior Homes
-                    </styles.TruncatedText>
-                  </li>
-                </styles.SettingListedItems>
+              <ul style={{ paddingLeft: '1.25rem', listStyleType: 'disc' }}>
```
Comment (jxmoose) [inline components/SettingsCard/SettingsCardShowPreferences.tsx]: use the SettingListedItems styled component

File: components/SettingsCard/SettingsCardShowPreferences.tsx [hunk 5]
```diff
@@ -22,63 +22,60 @@ export default function SettingCardShowPreferences() {
...
-                    <styles.TruncatedText
-                      $fontWeight="400"
-                      $color={COLORS.gray11}
-                      $align="left"
-                    >
-                      Oakland, CA
-                    </styles.TruncatedText>
-                  </li>
-                </styles.SettingListedItems>
+              <ul style={{ paddingLeft: '1.25rem', listStyleType: 'disc' }}>
```
Comment (jxmoose) [inline components/SettingsCard/SettingsCardShowPreferences.tsx]: use the SettingListedItems styled component

File: components/SettingsCard/SettingsCardShowPreferences.tsx [hunk 6]
```diff
@@ -22,63 +22,60 @@ export default function SettingCardShowPreferences() {
...
-                    <styles.TruncatedText
-                      $fontWeight="400"
-                      $color={COLORS.gray11}
-                      $align="left"
-                    >
-                      Seniors
-                    </styles.TruncatedText>
-                  </li>
-                </styles.SettingListedItems>
+              <ul style={{ paddingLeft: '1.25rem', listStyleType: 'disc' }}>
```
Comment (jxmoose) [inline components/SettingsCard/SettingsCardShowPreferences.tsx]: use the SettingListedItems styled component

File: package.json
Comment (jxmoose) [inline package.json:1]: remove these changes
